src/patternfly/components/DescriptionList/description-list.scss

```diff
@@ -0,0 +1,121 @@
+$pf-c-description-list--breakpoint-map: build-breakpoint-map("base", "md", "lg", "xl", "2xl");
+
+.pf-c-description-list {
+  --pf-c-description-list--RowGap: var(--pf-global--gutter--md);
+  --pf-c-description-list--ColumnGap: var(--pf-global--spacer--md);
+  --pf-c-description-list--GridTemplateColumns--count: 1;
+  --pf-c-description-list--GridTemplateColumns--width: 1fr;
+  --pf-c-description-list--GridTemplateColumns: repeat(var(--pf-c-description-list--GridTemplateColumns--count), var(--pf-c-description-list--GridTemplateColumns--width));
+
+  // Group
+  --pf-c-description-list__group--RowGap: var(--pf-global--spacer--sm);
+  --pf-c-description-list__group--ColumnGap: var(--pf-global--spacer--md);
+  --pf-c-description-list__group--GridTemplateColumns: auto;
+  --pf-c-description-list__group--GridColumn: auto;
+  --pf-c-description-list__group--AlignItems: baseline;
+
+  // Header
+  --pf-c-description-list__header--GridColumn: 1 / -1;
+
+  // Term
+  --pf-c-description-list__term--FontWeight: var(--pf-global--FontWeight--bold);
+  --pf-c-description-list__term--FontSize: var(--pf-global--FontSize--sm);
+
+  // Description
+  --pf-c-description-list__description--FontWeight: var(--pf-global--FontWeight--regular);
+  --pf-c-description-list__description--FontSize: var(--pf-global--FontSize--md);
+
+  // Horizontal
+  --pf-c-description-list--m-horizontal__term--width: #{pf-size-prem(150px)};
+  --pf-c-description-list--m-horizontal__description--width: minmax(#{pf-size-prem(150px)}, auto);
```

Comment:
should this be
```suggestion
  --pf-c-description-list--m-horizontal__description--width: minmax(var(--pf-c-description-list--m-horizontal__term--width), auto);
```

Reply:
I don't think so, although it could be. It's not a breaking change if we update next sprint.